OpenXmlPowerTools/DocumentBuilder.cs

```diff
@@ -727,6 +698,9 @@ private static void InitEmptyHeaderOrFooter(MainDocumentPart mainDocPart, XEleme
                         </w:r>
                       </w:p>
                     </w:ftr>");
+
+                NamespaceAttributeUtil.ApplyNamespaceAttributes(xDoc.Root);
+
```

Comment:
You can replace all of the above with this:
```
xDoc = new XDocument(
    new XDeclaration("1.0", "utf-8", "yes"),
    new XElement(W.ftr, NamespaceAttributes,
        new XElement(W.p,
            new XElement(W.pPr,
                new XElement(W.pStyle, new XAttribute(W.val, "Footer"))))));
```
A `w:t` with a quasi-empty `w:t` is not required (as above).